timescaledb/init.sql

```diff
@@ -0,0 +1,6 @@
+-- set up the grafanareader role with select permissions by default
+CREATE ROLE grafanareader;
+ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO grafanareader;
```

Comment:
I think it would be better to use the same permissions here as the production version does, so we can pick up on any potential issues earlier. The SQL statement for doing this is [here](https://github.com/ebmdatalab/metrics/blob/main/grafana/README.md#configure-user-on-postgresql-db-cluster).

Reply:
That's the permissions for connecting to the jobserver read-replica, this change is for connecting to the timescaledb!